.changeset/old-bulldogs-fry.md

```diff
@@ -0,0 +1,16 @@
+---
+'@backstage/plugin-catalog-backend-module-github': patch
+---
+
+Handle Github `github.push` events at the `GithubEntityProvider` by subscribing to the topic `github.push.`
```

Comment:
```suggestion
Handle GitHub `push` events at the `GithubEntityProvider` by subscribing to the topic `github.push.`
```
The webhook event itself is just called `push` and we receive them at an endpoint for the topic `github` and an event router re-publishes it for topic `github.push` based on the metadata.